prepare/cards/safety/simple_safety_tests.py

```diff
@@ -0,0 +1,35 @@
+from unitxt import add_to_catalog
+from unitxt.blocks import (
+    LoadHF,
+    TaskCard,
+)
+from unitxt.operators import (
+    AddFields,
+    RenameFields,
+)
+from unitxt.test_utils.card import test_card
+
+card = TaskCard(
+    loader=LoadHF(
+        path="Bertievidgen/SimpleSafetyTests", data_classification_policy=["public"]
+    ),
+    preprocess_steps=[
+        RenameFields(field_to_field={"prompt": "input"}),
+        AddFields(
+            fields={
+                "output": "None",
+                "type_of_input": "question",
+                "type_of_output": "answer",
+                "prediction_type": "str",
+            }
+        ),
+    ],
+    task="tasks.generation",
```

Comment:
In the full version of this test the model generates a list of violated harm categories in addition to the safe/unsafe answer. The rationale was it will be easier to parse with `tasks.generation`.
cc: @OfirArviv - right?

Reply:
yes. Also, we will need to streamline the llm-as-judge usage of tasks and such anyway. So this is for now.